src/php/install.sh

```diff
@@ -11,17 +11,17 @@ set -eux
 # Clean up
 rm -rf /var/lib/apt/lists/*
 
-VERSION=${VERSION:-"latest"}
-INSTALL_COMPOSER=${INSTALLCOMPOSER:-"true"}
-OVERRIDE_DEFAULT_VERSION=${OVERRIDEDEFAULTVERSION:-"true"}
+VERSION="${VERSION:-"latest"}"
+INSTALL_COMPOSER="${INSTALLCOMPOSER:-"true"}"
+OVERRIDE_DEFAULT_VERSION="${OVERRIDEDEFAULTVERSION:-"true"}"
 
-export PHP_DIR=${PHP_DIR:-"/usr/local/php"}
-USERNAME=${USERNAME:-"automatic"}
-UPDATE_RC=${UPDATE_RC:-"true"}
+export PHP_DIR="${PHP_DIR:-"/usr/local/php"}"
+USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
+UPDATE_RC="${UPDATE_RC:-"true"}"
 
 # Comma-separated list of php versions to be installed
 # alongside VERSION, but not set as default.
-ADDITIONAL_VERSIONS=${ADDITIONALVERSIONS:-""}
```

Comment:
https://github.com/devcontainers/features/pull/309/commits/627fc411904a0a68d7e3d9cc6a7af79609d57720 Making similar changes should fix failing the php:ubuntu:bionic test.

Reply:
We could fix the test in another PR 👍

Reply:
Yeah, and the Java one has this error in it for `install_from_non_default_distro`:

```
#16 0.950 find: Failed to restore initial working directory: /tmp/build-features/java_4: Permission denied
#16 0.961 find: Failed to restore initial working directory: /tmp/build-features/java_4: Permission denied
#16 0.963 find: Failed to restore initial working directory: /tmp/build-features/java_4: Permission denied
```

Which again seems like a general bug.